index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Registeration Form</title>
+    
+    <link rel="stylesheet" href="Styles.css">
```

Comment:
In HTML, kindly rename this to match your stylesheet "style.css" so that your css file would be linked to your html file